### PR TITLE
feat(manifests): ODH Dashboard tiles

### PR DESCRIPTION
**Pull request by gmfrasca**
**Description of your changes:**
Adds an OdhApplication and OdhQuickstart CR which can be deployed as an overlay

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: manifests/opendatahub/overlays/integration-odhdashboard/odhapplications/data-science-pipelines-odhapplication.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - [Using Tekton Custom Task on DSP](https://github.com/opendatahub-io/ml-pipelines/tree/master/samples/tekton-custom-task)
+    - [The flip-coin pipeline using custom task](https://github.com/opendatahub-io/ml-pipelines/tree/master/samples/flip-coin-custom-task)
+    - [Retrieve DSP run metadata using Kubernetes downstream API](https://github.com/opendatahub-io/ml-pipelines/tree/master/samples/k8s-downstream-api)
+
+  internalRoute: ml-pipeline-ui
+  kfdefApplications:
+  - data-science-pipelines
+  provider: Red Hat
+  quickStart: create-data-science-pipeline
+  route: ml-pipeline-ui
```
Comment (gmfrasca) [inline manifests/opendatahub/overlays/integration-odhdashboard/odhapplications/data-science-pipelines-odhapplication.yaml]: note: may need to change this based on [odh-manifests change](https://github.com/red-hat-data-services/odh-manifests/pull/245)

Comment (anishasthana) [reply manifests/opendatahub/overlays/integration-odhdashboard/odhapplications/data-science-pipelines-odhapplication.yaml]: We need to make the corresponding upstream change too.

**Review by anishasthana — COMMENTED** [review]
I'd say we should raise this PR after the renaming of everything, but I'm not opposed to merging it in now and making sure the rename PR tracks this update.

**Review by anishasthana — APPROVED** [review]
/lgtm

**Comment by anishasthana** [comment]
Merging this with the team's blessings due to time constraints.

**Pull request merged**